lib/autoupdate/agent/setup.go

```diff
@@ -273,11 +289,198 @@ func (ns *Namespace) Setup(ctx context.Context, path string) error {
 			}
 		}
 	}
+
+	return nil
+}
+
+type filePaths struct {
```

Comment:
Maybe define this inline? Or in the selinux package?

Reply:
defined in selinux package